Cargo.toml

```diff
@@ -33,7 +33,7 @@ alloy-transport-http = { git = "https://github.com/alloy-rs/alloy.git", rev = "5
 aws-config = "1.1.7"
 aws-sdk-kms = "1.15.0"
 
-hex = "0.4.3"
+hex = { package = "const-hex", version = "1.10", default-features = false, features = ["alloc"] }
```

Comment:
note: switched to const-hex here, as alloy uses it. This is equivalent to `hex` but much, much faster, and maintained by Dani (`hex` is largely unmaintained).